sui-execution/latest/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -831,6 +831,7 @@ fn vm_move_call(
     Ok(result)
 }
 
+#[allow(clippy::extra_unused_type_parameters)]
 fn deserialize_modules<Mode: ExecutionMode>(
```

Comment:
huh... I think we could get rid of this type parameter, but I'm happy to do that as a follow-up.